docs/Migrating to Newer SDL Versions/Updating to v4.7/index.md

```diff
@@ -582,7 +582,7 @@ public void onOnButtonEvent(OnButtonEvent notification) {
    Log.i(TAG, "onOnButtonEvent: ");
 
    if (notification.getButtonName() == CUSTOM_BUTTON){
-        int ID = notification.getCustomButtonName();
+        int ID = notification.getCustomButtonID();
```

Comment:
I don't think the method name in the migration guide for 4.7 should be updated because 4.7 doesn't have the new getCustomButtonID() method.
```suggestion
        int ID = notification.getCustomButtonName();
```

Reply:
Updated